src/util/StringUtils.cpp

```diff
@@ -48,8 +48,15 @@ bool strIsLangTag(const string& input) {
 }
 
 // ____________________________________________________________________________
-bool isLangMatch(std::string_view strLangTag, std::string_view strLangRange) {
-  return strLangTag.starts_with(strLangRange);
+bool isLanguageMatch(string& langTag, string& langRange) {
+  if (langRange.ends_with("*")) {
```

Comment:
I think the case that one of them is empty can also be implemented here, currently it is
mixed betweeen here and the actual expression.